psychopy/experiment/components/button/__init__.py

```diff
@@ -275,6 +275,7 @@ def writeFrameCode(self, buff):
         indented = self.writeStartTestCode(buff)
         if indented:
             code = (
+                "%(name)s.buttonClock.reset()"
```

Comment:
the clock for the button should surely be set during the win.flip, shouldn't it? When you set autodraw to be true the button is not visible. I think this should be a callOnFlip call

Reply:
Good call, just made the change